rust-toolchain.toml

```diff
@@ -1,3 +1,5 @@
 [toolchain]
 channel = "nightly-2021-12-15"
+components = ["rustfmt"]
 targets = ["wasm32-unknown-unknown"]
+profile = "minimal"
```

Comment:
I don't fully know the implications of this. Looks like minimal is the right choice for CI but setting it here will also affect our local development.

Could explicitly set the profile for CI in the action
```
rustup set profile minimal
```

Reply:
Was referencing this doc- https://rust-lang.github.io/rustup/concepts/profiles.html